Resources/Prototypes/Entities/Structures/plastic_flaps.yml

```diff
@@ -0,0 +1,65 @@
+- type: entity
+  id: PlasticFlapsClear
+  parent: BaseStructureDynamic
+  name: plastic flaps
+  suffix: Clear
+  description: Heavy duty, plastic flaps. Definitely can't get past those. No way.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Structures/plastic_flaps.rsi
+    state: plasticflaps
+  - type: Physics
+  - type: Transform
+    anchored: true
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+        bounds: "-0.49,-0.49,0.49,0.49"
+      mass: 100
+      mask:
+      - Passable
```

Comment:
Please don't ever use this mask.

Reply:
yeah, removed this one, had it working and had thought it was part of the reason.